copy/copy.go

```diff
@@ -577,6 +577,29 @@ func (c *copier) copyOneImage(ctx context.Context, policyContext *signature.Poli
 		}
 	}
 
+	destExistsCheck := true // TBD: enable configuration
+	if destExistsCheck {
+		destImageSource, err := c.dest.Reference().NewImageSource(ctx, options.DestinationCtx)
+
+		if err == nil {
+
+			retManifest, retManifestType, err = destImageSource.GetManifest(ctx, nil)
+			if err != nil {
+				return nil, "", "", errors.Wrapf(err, "Error reading manifest from dest image %s", transports.ImageName(destImageSource.Reference()))
+			}
+
+			sourceManifest, _, err := src.Manifest(ctx)
+			if err != nil {
+				return nil, "", "", errors.Wrapf(err, "Error reading manifest from source image")
+			}
+
+			if bytes.Compare(sourceManifest, retManifest) == 0 {
+				c.Printf("Dest image manifest compares equal\n")
```

Comment:
Soft suggestion here and above for dest -> destination
```suggestion
				c.Printf("Destination image manifest compares equal\n")
```

Reply:
Thanks! I've also amend my initial PR note with a sample output